stack/globals_test.go

```diff
@@ -2795,54 +2796,6 @@ func TestLoadGlobals(t *testing.T) {
 				),
 			},
 		},
-		{
```

Comment:
tests moved to mapexpr/test/testcase.go so they could be reused with `lets` (with both genfile and genhcl).